From 794ac6fcc0854f08a059de2708df541fb9a4ec75 Mon Sep 17 00:00:00 2001
From: Nabi <artsanaat@gmail.com>
Date: Mon, 13 Nov 2023 14:40:23 +0330
Subject: [PATCH] Fixed deprecation warnings in plugin
 wpb-woo-product-slider-pro due to required parameters following optional
 parameters.

---
 admin/metaboxes/meta_box.php | 2 +-
 inc/wpb_wps_functions.php    | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/admin/metaboxes/meta_box.php b/admin/metaboxes/meta_box.php
index c8b444376..3c5e92225 100644
--- a/admin/metaboxes/meta_box.php
+++ b/admin/metaboxes/meta_box.php
@@ -413,7 +413,7 @@ if( !function_exists('wpb_wps_custom_meta_box') ){
 		 *
 		 * @return	bool				whether or not the type is in the provided array
 		 */
-		function wpb_wps_meta_box_find_repeatable( $needle, $haystack ) {
+		function wpb_wps_meta_box_find_repeatable( $needle, $haystack = array() ) {
 			foreach ( $haystack as $h )
 				if ( isset( $h['type'] ) && $h['type'] == $needle )
 					return true;
diff --git a/inc/wpb_wps_functions.php b/inc/wpb_wps_functions.php
index da93ed28a..656df554d 100644
--- a/inc/wpb_wps_functions.php
+++ b/inc/wpb_wps_functions.php
@@ -552,7 +552,7 @@ add_filter( 'product_cat_row_actions', 'wpb_wps_show_term_id', 10, 2 );
 add_filter( 'product_tag_row_actions', 'wpb_wps_show_term_id', 10, 2 );
 
 function wpb_wps_show_term_id(  $actions, $term ){
-
+    $actions = empty( $actions ) ? array() : $actions;
 	$actions['id'] = esc_html__( 'ID: ', 'wpb-woocommerce-product-slider' ) . esc_html( $term->term_id );
 
 	return $actions;
-- 
2.15.0

